doc/cmake-vols-fetchcontent.md

```diff
@@ -49,26 +52,44 @@ For each URL specified, HDF5's CMake code will attempt to use CMake's
 [FetchContent](https://cmake.org/cmake/help/latest/module/FetchContent.html)
 functionality to retrieve the source code for a VOL connector pointed to by
 that URL and will try to build that VOL connector as part of the HDF5 library
-build process. The VOL connector must be able to be built by CMake and currently
+build process. 
+
+If `HDF5_ALLOW_EXTERNAL_SUPPORT` is set to "LOCAL_DIR", then the CMake cache 
```

Comment:
Please see the existing use of HDF5_ALLOW_EXTERNAL_SUPPORT; these changes are overwriting current usage reqs.